src/hooks/core/websocket/types.d.ts

```diff
@@ -41,6 +37,31 @@ export type GetstreamMessage = {
     updated_at: string;
     username: string;
   };
+  /**
+   * SYSTEM MESSAGE CUSTOM PARAM
+   */
+  system_user?: string;
+  other_system_user?: string;
+  isSystem?: boolean;
+  ignore_update_timestamp?: boolean;
+  ignore_unread_count?: boolean;
+  textOwnMessage?: string;
+  type: string;
+  better_type?:
+    | 'follow_user'
+    | 'follow_topic'
+    | 'new_topic_post'
+    | 'add_member_to_group'
+    | 'remove_member_from_group'
+    | 'change_channel_detail'
+    | 'leave_group';
+  only_show_to_system_user?: boolean;
+  text: string;
+  other_text?: string;
+  own_text?: string;
+  /**
+   * SYSTEM MESSAGE END
+   */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of these new fields in the `GetstreamMessage` type seems to be for handling system messages. However, it's not clear what each field represents just from their names. It would be beneficial to add comments explaining the purpose of each field for better maintainability and understanding of the code by other developers.
```diff
  /**
   * SYSTEM MESSAGE CUSTOM PARAM
   */
  system_user?: string; // Add comment here
  other_system_user?: string; // Add comment here
  isSystem?: boolean; // Add comment here
  ignore_update_timestamp?: boolean; // Add comment here
  ignore_unread_count?: boolean; // Add comment here
  textOwnMessage?: string; // Add comment here
  type: string;
  better_type?:
    | 'follow_user'
    | 'follow_topic'
    | 'new_topic_post'
    | 'add_member_to_group'
    | 'remove_member_from_group'
    | 'change_channel_detail'
    | 'leave_group';
  only_show_to_system_user?: boolean; // Add comment here
  text: string;
  other_text?: string; // Add comment here
  own_text?: string; // Add comment here
  /**
   * SYSTEM MESSAGE END
   */
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->